Resources/Prototypes/_ES/Roles/Jobs/Command/head_of_personnel.yml

```diff
@@ -0,0 +1,74 @@
+- type: job
+  id: ESHeadOfPersonnel
+  name: job-name-hop
+  description: job-description-hop
+  playTimeTracker: ESJobHeadOfPersonnel
+  requirements:
+    - !type:DepartmentTimeRequirement
+      department: Engineering
+      time: 2.5h
+    - !type:DepartmentTimeRequirement
+      department: Medical
+      time: 2.5h
+    - !type:DepartmentTimeRequirement
+      department: Science
+      time: 2.5h
+    - !type:DepartmentTimeRequirement
+      department: Security
+      time: 2.5h
+    - !type:DepartmentTimeRequirement
+      department: Command
+      time: 2.5h
+  weight: 20
+  startingGear: ESHoPGear
+  icon: "JobIconHeadOfPersonnel"
+  supervisors: job-supervisors-captain
+  canBeAntag: false
+  access:
+  - Command
+  - HeadOfPersonnel
+  - Bar
+  - Service
+  - Maintenance
+  - Janitor
+  - Theatre
+  - Kitchen
+  - Chapel
+  - Hydroponics
+  - External
+  - Cryogenics
+  # I mean they'll give themselves the rest of the access levels *anyways*.
+  # As of 15/03/23 they can't do that so here's MOST of the rest of the access levels.
+  # Head level access that isn't their own was deliberately left out, get AA from the captain instead.
```

Comment:
i dont remember what they're missing but HOP should probably just have all access except captain and HOS. they don't really have anything they're competing with anymore.